MapboxCoreNavigation/RouteController.swift

```diff
@@ -165,6 +212,78 @@ open class RouteController: NSObject {
         locationManager.stopUpdatingLocation()
         locationManager.stopUpdatingHeading()
     }
+    
+    /**
+     Send feedback about the current road segment/maneuver to the Mapbox data team.
+     
+     You can pair this with a custom feedback UI in your app to flag problems during navigation
+     such as road closures, incorrect instructions, etc. 
+     
+     With the help of a custom `description` to elaborate on the nature of the problem, using
+     this function will automatically flag the road segment/maneuver the user is currently on for 
+     closer inspection by Mapbox's system and team.
+     
+     If you provide a custom feedback UI that lets users elaborate on an 
+     issue, you should call this before you show the custom UI so the 
+     location and timestamp are more accurate. You can then call 
+     `updateLastFeedback()` to attach any additional metadata to the 
+     feedback.
+     */
+    public func recordFeedback(type: FeedbackType, description: String?) {
```

Comment:
Eventually, we may want to change this method to take a fully formed Feedback instance, in order to give the developer more fields they can fill out but avoid proliferating arguments.